src/webpack.config.js

```diff
@@ -32,11 +33,24 @@ module.exports = {
 		],
 		loaders: [
 			{ test: /src[\\\/].*\.ts?$/, loader: 'umd-compat-loader!ts-loader' },
-			{ test: /\.js?$/, loader: 'umd-compat-loader' },
+			{ test: /\.js?$/, loader: 'umd-compat-loader'  },
 			{ test: /\.html$/, loader: 'html' },
 			{ test: /\.(jpe|jpg|png|woff|woff2|eot|ttf|svg)(\?.*$|$)/, loader: 'file?name=[path][name].[hash:6].[ext]' },
-			{ test: /\.styl$/, loader: ExtractTextPlugin.extract(['css-loader?sourceMap', 'stylus-loader']) },
-			{ test: /\.css$/, loader: 'style-loader!css-loader?modules' },
+			{
+				test: /\.styl$/,
+				exclude: /\.module\.styl$/,
+				loader: ExtractTextPlugin.extract([
+					'css-loader?sourceMap',
+					'stylus-loader'
+				])
+			},
+			{
+				test: /\.module\.styl$/,
+				loader: ExtractTextPlugin.extract([
+					`css-loader?modules&sourceMap&importLoaders=1&localIdentName=${cssModuleIdent}`,
```

Comment:
I don't think we need `importLoaders=1` here. Believe that may have came from my advice previously but please let me know otherwise.